web/controllers/admin_controller.ex

```diff
@@ -319,6 +319,7 @@ defmodule ExAdmin.AdminController do
   end
 
   defp to_integer(:string, string), do: string
+  defp to_integer(:id, string), do: string
```

Comment:
Hi Steve,
I am not so sure that we should attempt convert id to integer. To be honest I just started trying ex_admin today.
Today after installing and I was trying it. At first try it feel nice trying it. And I came to this error eventhough my model primary was integer. So I forked it and try to trace the error. And I see that the match wasn't matched so I added line and everything start working. 

If my guess is correct this pattern match is used to change the parameter to integer or string and then will use it as primary key on the batch query.

From what I see ecto allows integer , uuid, string (possibly another type also I am not so sure.#custom primary key) As a primary key. So I think we should just check for primary key type and then we just cast it to that type to make it generic case. 

Anyway I would like to contribute to this nomatter my guess is correct or not. Please let's me know if I can help contribute to this project. I am going try and play with it.

Thanks
Dev